ksql-cli/src/main/java/io/confluent/ksql/util/CliUtils.java

```diff
@@ -155,7 +154,7 @@ public static boolean createFile(Path path) {
       }
       return true;
     } catch (Exception e) {
-      LOGGER.error(ExceptionUtil.stackTraceToString(e));
+      LOGGER.error("createFile failed, path:" + path, e);
```

Comment:
is this an error or should it be warn?

Reply:
also here and everywhere else in the log statements we should use "{}" for args rather than string concatentation

Reply:
done